pkg/controller/jobs/jobset/jobset_webhook.go

```diff
@@ -58,6 +59,20 @@ func (w *JobSetWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	jobSet := fromObject(obj)
 	log := ctrl.LoggerFrom(ctx).WithName("jobset-webhook")
 	log.V(5).Info("Applying defaults", "jobset", klog.KObj(jobSet))
+
+	if wlName, hasPrebuilt := jobframework.PrebuiltWorkload(jobSet); hasPrebuilt {
+		for i := range jobSet.Spec.ReplicatedJobs {
+			rjTemplate := &jobSet.Spec.ReplicatedJobs[i].Template
+			if rjTemplate.Annotations == nil {
+				rjTemplate.Annotations = map[string]string{
+					constants.ParentWorkloadAnnotation: wlName,
```

Comment:
just initialize the map in this branch to empty, and set the value outside else. I think it would read better.

Reply:
done